tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -106,6 +106,15 @@ def kill_all(self):
         self.marionette.switch_to_frame()
         self.marionette.execute_async_script("GaiaApps.killAll();")
 
+    def uninstall(self, name):
+        self.marionette.switch_to_frame()
+        data_layer = GaiaData(self.marionette)
+        data_layer.set_bool_pref('dom.mozApps.auto_confirm_uninstall', True)
+        result = self.marionette.execute_async_script('GaiaApps.uninstallWithName("%s")' % name)
+        if not result is True:
```

Comment:
Why don't we use `assert` like done in [kill](https://github.com/mozilla-b2g/gaia/pull/29550/files#diff-6d544bbb7c759ffb68f344f348f7740eL96) just above?